web/src/containers/admin/EditAccount.js

```diff
@@ -1,115 +1,112 @@
+import { FormLabel, Select, TextField } from '@cmsgov/design-system-core';
 import PropTypes from 'prop-types';
 import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
-import { Link } from 'react-router-dom';
-import { STATES } from '../../util';
 
 import { editAccount as editAccountDispatch } from '../../actions/admin';
-import Btn from '../../components/Btn';
-import { t } from '../../i18n';
+import CardForm from '../../components/CardForm';
+import Header from '../../components/Header';
+import { getEditAccountError } from '../../reducers/errors';
+import { getEditAccountWorking } from '../../reducers/working';
+import { STATES } from '../../util';
 
 class EditAccount extends Component {
   state = {
+    hasFetched: false,
+    success: false,
     userID: '',
     user: null
   };
 
+  static getDerivedStateFromProps({ error, working }, { hasFetched }) {
+    if (!hasFetched) {
+      return { hasFetched: working };
+    }
+
+    return {
+      success: !working && !error
+    };
+  }
```

Comment:
This one is simpler than the last because the user comes from the application state rather than the component logic.